st2common/st2common/models/api/execution.py

```diff
@@ -105,6 +105,22 @@ class ActionExecutionAPI(BaseAPI):
                 "type": "array",
                 "items": {"type": "string"},
                 "uniqueItems": True
+            },
+            "log": {
```

Comment:
If we only store status information in this attribute, I would prefer to have  some more self-explanatory name (status_transition_log or similar).

If we plan to expand it later, I'm fine with this name though.